Makefile

```diff
@@ -1,12 +1,16 @@
-.PHONY: unit e2e-test
+.PHONY: test
 
 test-unit:
+	@echo "Installing test deps..."
+	pip install -r requirements-test.txt
```

Comment:
i think you need to pip install requirements.txt too

Reply:
Fixed.

Reply:
thinking about this more, i don't think we want this make target to install any deps, similar to my reasoning here:  https://github.com/openshift/lightspeed-service/pull/21/files#r1409664767

(it also takes additional time, even when the deps are already installed, which will be annoying to local developers).

i think we should have a couple more make targets:
`make install-deps`
`make install-test-deps`

and then either another target invokes the install-deps/test-deps targets and the test-unit target, or something like that.

but we can follow up w/ that i guess.